### ISSUE TITLE
install config 404 error

### ISSUE THREAD

**Issue by omni360** [gogs/gogs#49]
can't found the config page

**Comment by letompouce** [comment]
tl;dr: fill the install form, click `Install`, disregard the « 404 » error because everything went fine anyway.

I followed the instructions for the [Debian 8 package install](https://packager.io/gh/pkgr/gogs/install?bid=418#debian-8-gogs), with `gogs 0.8.25-1454711359.e797a22.jess amd64` using MySQL as backend.

First connection to gogs, got the install page ; fill the forms, hit the `Install` button, got a 404 not found image. I had to leave for some time, when I got back I hit `Previous` but somehow the session expired or something, so I tried to open the main URL (localhost:3000) to get back to the install process, but then, the expected UI appears, like the install process completed anyway.
